.clang-format

```diff
@@ -34,7 +34,7 @@ AlignConsecutiveBitFields:
   AlignFunctionPointers: false
   PadOperators:    false
 AlignConsecutiveDeclarations:
-  Enabled:         true
```

Comment:
Thanks @dzenanz By the way, why does Examples have its own ".clang-format" file? Is it intended to be an exact copy of the ".clang-format" file in the root? It now has different values for `ColumnLimit` and `PenaltyBreakFirstLessLess`.

Reply:
@N-Dekker The Examples directory uses the same settings as the parent directory, EXCEPT for the line length limits.  Examples need a really short line limit because those lines get rendered into the software guide, and when rendered with LaTeX, they need to be much shorter to fit on the page.

Reply:
If you edit the Examples/.config, would you mind putting the above explanation into to comments at the top of the file?

Thanks,
Hans

Reply:
@hjmjohnson Does the `.clang-format` in Examples directory need to have all the settings from the root? Can we only override the settings that are different (related to line lengths)?

Reply:
> only override the settings that are different

@N-Dekker this should not be hard to try.

Reply:
Thanks for your explanation and discussion, @dzenanz and @hjmjohnson For this PR I just propose to toggle the AlignConsecutiveDeclarations option. For other possible improvements (specifically for the ".clang-format" file in the Examples, I would prefer to see a separate PR. Because this PR (5096) will already become quite large, once it has all the code formatting changes included.

Reply:
@N-Dekker I'd rather see the Examples changes to that flag kept in the same PR.  Examples are critical to the documentation.  I'm OK if it's a separate commit, but the proposed changes should e adjacent commits in the history.

Reply:
[force-push](https://github.com/InsightSoftwareConsortium/ITK/compare/3213446b46a6fa4e549472e3ffce26483a87ef80..4c969431da302e4510ffa285a6cdf9e4bd4f8f7a) just to address the initial comment by @dzenanz at https://github.com/InsightSoftwareConsortium/ITK/pull/5096#discussion_r1892250025